Scheduler/observing_blocks_maintel/AOS/WEP/BLOCK-T390.json

```diff
@@ -0,0 +1,35 @@
+{
+    "name": "BLOCK-T390",
+    "program": "BLOCK-T390",
+    "constraints": [],
+    "configuration_schema": "$schema: http://json-schema.org/draft-07/schema#\ntitle: BLOCK-T390 Configuration\ndescription: Configuration for the BLOCK-T390 Wavefront Estimation Chromaticity Test WET-008.\ntype: object\nproperties:\n  ra:\n    description: Right Ascension.\n    type: number\n    default: 0\n  dec:\n    description: Declination.\n    type: number\n    default: 0\n",
+    "scripts": [
+        {
+            "name": "maintel/track_target.py",
+            "standard": true,
+            "parameters": {
+                "slew_icrs": {
+                    "ra": "$ra", 
+                    "dec": "$dec"
+                }
+            }
+        },
+        {
+            "name": "maintel/take_image_lsstcam.py",
```

Comment:
I believe we should send a command that changes anything in the optical state before taking an image, right?